util/Setup/EnvironmentFileBuilder.cs

```diff
@@ -95,6 +95,8 @@ private void Init()
                 ["globalSettings__attachment__baseUrl"] = $"{_context.Config.Url}/attachments",
                 ["globalSettings__dataProtection__directory"] = $"{_context.OutputDir}/core/aspnet-dataprotection",
                 ["globalSettings__logDirectory"] = $"{_context.OutputDir}/logs",
+                ["globalSettings__logRollBySizeLimit"] = string.Empty,
+                ["globalSettings__Syslog__Destination"] = string.Empty,
```

Comment:
Following convention should use camelCase vs. PascalCasing, i.e. `["globalSettings__syslog__destination"]`

Reply:
I agree, but #742 uses `Syslog.Destination` in its code...

Reply:
Yeah, I'm not entirely sure how that mapping works, but `globalSettings.LogRollBySizeLimit` is also in the code, and the casing just "works".

Reply:
Case just updated then @cscharf, thank you for your review 👍